Hymns/Home/LaunchRouterView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct LaunchRouterView: View {
+    @State var showSplash: Bool = false
+
+    var body: some View {
+        Group { () -> AnyView in
+            if showSplash {
+                return SplashScreenView()
+                    .opacity(showSplash ? 1 : 0)
```

Comment:
Do we need this opacity? If showSplash is false, we won't even show this right

Reply:
Nice, good catch.